import_mu.py

```diff
@@ -29,7 +29,7 @@
 
 from mu import MuEnum, Mu, MuColliderMesh, MuColliderSphere, MuColliderCapsule
 from mu import MuColliderBox, MuColliderWheel
-from shader import make_shader
+from shader import make_material
```

Comment:
`make_shader` was removed for `make_material` since now, we no longer deal with shader nodes but classic materials. (We talked about that with tigrou and since shader nodes are deprecated, we don't really have to deal with this structure)